### ISSUE TITLE
Invoking Sonarqube using generated VS IDE is slower due to multiple Gradle invocation

### ISSUE THREAD

**Issue by khickel** [nokeedev/gradle-native#265]
Using SonarQube to scan C/C++ code from the command line, such as in a daily build from Jenkins requires the use of their build-wrapper.  The build-wrapper does not detect the compiler/linker invocation from gradle, and there isn't any documentation that would help to understand why.

When building a generated visual studio project, either from the VS IDE, or from MSBuild, the build of each project invokes gradle, which represents a lot of overhead.

**Comment by lacasseio** [comment]
I don't believe running the build-wrapper on MSBuild using the VS solution generated by Nokee is the right path. The execution chain would be as follow: `build-wrapper` -- monitor --> `MSBuild` -- delegate --> `Gradle/Nokee`. There are several "middle" layer that just adds complexity and failure point. The generated VS Solution/Project from Nokee is close to vanilla but we do overwrite some key tasks which probably confuses the build-wrapper. You can see the result [here](https://sonarcloud.io/dashboard?id=nokeedev_nokee-samples-sonarqube).

I wrote a [demonstration](https://github.com/nokeedev/nokee-samples-sonarqube) that improve upon the [official Gradle plugin](https://github.com/SonarSource/sonar-scanner-gradle). I also [kick off the discussion for a feature request](https://community.sonarsource.com/t/support-for-native-ecosystem-in-gradle-plugin/39295) in the official plugin.

Try the plugin in the demonstration and tell me if that solve your issue.
